src/main/java/com/exasol/projectkeeper/validators/pom/plugin/ReproducibleBuildPluginValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.exasol.projectkeeper.validators.pom.plugin;
+
+import java.util.Collection;
+import java.util.function.Consumer;
+
+import org.w3c.dom.Node;
+
+import com.exasol.projectkeeper.ProjectKeeperModule;
+import com.exasol.projectkeeper.ValidationFinding;
+
+/**
+ * Validator for reproducible-build-maven-plugin.
+ */
+public class ReproducibleBuildPluginValidator extends AbstractPluginPomValidator {
+
+    /**
+     * Create a new instance of {@link ReproducibleBuildPluginValidator}.
+     */
+    public ReproducibleBuildPluginValidator() {
+        super("maven_templates/reproducible-build-maven-plugin.xml");
+    }
+
+    @Override
+    public ProjectKeeperModule getModule() {
+        return ProjectKeeperModule.MAVEN_CENTRAL;
```

Comment:
It should be a default mode, the plugin is required with GitHub release too.

Reply:
you're right